### ISSUE TITLE
Cockatrice Icon over Ubuntu windows

### ISSUE THREAD

**Issue by dco023** [Cockatrice/Cockatrice#1922]
<b>OS:</b>
Ubuntu 14.04 (Trusty Thar)

<b>Cockatrice version:</b>
 8d67c75 (2016-04-08) self-compiled

There's is a small cockatrice icon over all other windows at the top left corner while the software is running. It makes it impossible to minimize full sized windows. Is there any way around it?

You can see it at the screen shot I took. It's on the left top corner. It prevents clicking what's underneath it.

![screenshot from 2016-04-09 00 08 09](https://cloud.githubusercontent.com/assets/18359300/14401399/3a3c8a46-fde7-11e5-9f45-1299dad3d4e3.png)

**Comment by ctrlaltca** [comment]
Yes, this is a known problem on 14.04, when compiling using the qt5.2.1 version provided by ubuntu.
The only solution is to update Qt to a more recent version, eg:
```
    sudo apt-add-repository ppa:beineri/opt-qt551-trusty
    sudo apt-get update -qq
    sudo apt-get install -y qt55base qt55imageformats qt55multimedia qt55tools qt55svg
```
and then recompile ensuring that cmake is picking up the libraries from ```/opt/qt55/lib``` instead of the ones provided by the system in /usr/.

The minimize/maximize/close buttons should appear when you mouseover the upper left area of the screen (a debatable UI choice), but the first one (close) can't be clicked anyway. As a workaround, right click the icon and choose "close" or close cockatrice from the menu.

**Comment by Daenyth** [comment]
Is this still a problem using the latest Qt5 version? Are you still on 14.04LTS? 16.04LTS is out now.

**Comment by ctrlaltca** [comment]
It's just a matter of using a recent combination of Qt5 (not the one provided by Ubuntu) and appmenu-qt5 package on 14.04. It's fixed on 16.04LTS:

![trice](https://cloud.githubusercontent.com/assets/1631111/15902785/06d465fa-2daa-11e6-9170-088c2f106fa0.png)

**Comment by Daenyth** [comment]
I feel fine in saying that there's no action we should take on this then - it's an ubuntu issue and resolved by upgrading.
